### DISCUSSION TITLE
v116 broken with uc and selenium-wire

### DISCUSSION THREAD

**Discussion by strukiii** [ultrafunkamsterdam/undetected-chromedriver#1500] [Q&A] [answered]
Hey, not sure if anyone else has also had their project break this past week.  Putting this here because the selenium-wire repo seems dead.  In the meantime, I have no other choice but to downgrade chrome versions; I'm still in the process of clicking sketchy download links to find a working solution there. 

I've used the driver download solution [here](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/1491) for getting the base uc working.  But, when using uc with selenium-wire, requests fail that are proxied through mitmproxy.  I'm only interested in requests from a single domain, so the page loads fine minus the requests from this specific domain which fail.

**Comment by gorsek1** [comment]
Yeah same for me. Forgot to disable chrome updates...

**Comment by jdholtz** [comment, marked as answer]
@strukiii could you try my PR [here](https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/1503)? I noticed that the desired_capabilities is no longer respected and fixed it. It fixed an insecure certificate error I was receiving so it may help you out.

Also, do still get an error with Selenium Wire if you downgrade to Selenium 4.9 and UC 3.1.7?

**Reply by strukiii** [reply]
I'll give it a try, thanks a lot for your work :)

**Reply by strukiii** [reply]
works great, thanks again sir

**Reply by jdholtz** [reply]
Great to hear. Thanks for testing it out

**Comment by ultrafunkamsterdam** [comment]
The PR is merged and version is updated as well in pypi

**Reply by jdholtz** [reply]
This is actually another issue in Selenium 4.10+ which removed the desired_capabilities. I proposed a fix in another PR for this issue which allows the desired_capabilities keyword for Selenium 4.10+. This is helpful in the case of Selenium wire, where it uses this keyword, but doesn’t seem to be maintained any longer
